<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UXSpecPage</class>
 <widget class="QWidget" name="UXSpecPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>User Experience &amp; Interface Design</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="reviewPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="reviewInstructionLabel">
         <property name="text">
          <string>Below is the consolidated UX/UI Specification draft, including personas, user journeys, and a style guide. Please review it, make any direct edits, and provide general feedback for refinement below.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="infoLabel">
         <property name="text">
          <string>Tip: You can edit this draft directly. Please apply final modifications here. For general feedback or questions that require an AI response, please use the text box below.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="specTextEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="feedbackLabel">
         <property name="text">
          <string>Feedback for Refinement (Optional):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="feedbackTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="refineButton">
           <property name="text">
            <string>Submit Feedback &amp;&amp; Refine</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="approveButton">
           <property name="text">
            <string>Approve Specification</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>